### PR TITLE
Publish to chromatic from beta candidate branch

### PR DESCRIPTION
**Pull request by prprabhu-ms**
# Why
Storybook deployment for bug bashing.

# How Tested
Prayers.

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

**Comment by prprabhu-ms** [comment]
The idea is to land this, and then cherry-pick the PR to `beta-candidate`. If all goes well, merging the cherry-pick PR into `beta-candidate` will trigger a `publish-chromatic` workflow and give us a storybook to play with.

File: .github/workflows/publish-chromatic.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@ on:
       - packages/react-composites/**
       - packages/communication-react/**
   push:
-    branches: [main]
+    branches: [main, beta-candidate]
```
Comment (JamesBurnside) [inline .github/workflows/publish-chromatic.yml:12]: my vote here would be we just use `release`
So this release branch would be: `release/1.0.0-beta.2`
Thoughts?

This, with small update to set the correct branchname and overwrite an existing branch with that name in the case we want to choose a different alpha candidate, would then be all we need to run to create new beta candidates: https://github.com/Azure/communication-ui-library/blob/main/.github/workflows/npm-release-bump.yml

Comment (prprabhu-ms) [reply .github/workflows/publish-chromatic.yml:12]: There are two workflows to consider:
* We chose a new alpha tag to create a new beta candidate off of
* We merge changes onto a beta candidate branch to fix critical bugs.

We want to automate setup for bug-bashing in both these cases.

**Comment by prprabhu-ms** [comment]
Quick experiment I didn't follow up on.

**Pull request closed without merging**